schema/mysql/upgrades/1.1.2.sql

```diff
@@ -0,0 +1 @@
+UPDATE icingadb_schema SET timestamp = UNIX_TIMESTAMP(timestamp / 1000) * 1000;
```

Comment:
I'd add something like `WHERE timestamp > 20000000000000000` here for two reasons:

1. While we don't have idempotency as a requirement for schema upgrades, they probably shouldn't corrupt the data when accidentally applied twice (most other upgrades would probably bail out with a "something already exists" error).
2. Especially as you changed past schema upgrade files as well, if you'd now apply them in order, there would be broken timestamps left in this table.

That value is far enough in the future (after dividing by 1000), there won't be problems any time soon:
```console
date -ud@20000000000000.0000
Thu 08 Oct 635744 11:33:20 UTC
```